c/include/libsbp/orientation.h

```diff
@@ -36,6 +36,17 @@ SBP_PACK_START
  * that time-matched RTK mode is used when the base station is moving.
  */
 #define SBP_MSG_BASELINE_HEADING 0x020F
+#define SBP_BASELINE_HEADING__FIX_MODE_MASK (0x7)
```

Comment:
any reason a double underscore is needed? I can see an argument for readability, but it doesn't work on my eyes 👀